README.md

```
@@ -1,2 +1,236 @@
+   > Refer to the [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/) for styling and formatting.
+>
+
+4. Navigate to the `docs-bi/en/` directory in the terminal and build the repository with the following command:
+
+    ```shell
+    mkdocs serve
+    ```
+
+   > **NOTE:**
+   >
+   >Refer to [Run MkDocs](#run) for more information.
```

markdownlint-cli2 (0.21.0)

[warning] Link fragments should be valid

(MD051, link-fragments)